examples/android/helloworld/app/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ add_library(libgpr STATIC IMPORTED)
 set_target_properties(libgpr PROPERTIES IMPORTED_LOCATION
   ${GRPC_BUILD_DIR}/libgpr.a)
 
+add_library(libaddress_sorting STATIC IMPORTED)
```

Comment:
Needing to do this is an indication of  the this CMakeLists.txt being flawed. 
In this particular case, I think here's what's wrong:
- you include grpc sources using add_subdirectory()  (that itself is one of the fair approaches)
- grpc's CMakeLists.txt defines grpc targets  (I think here they would be referred to as e.g. `gRPC::libgrpc`), but you are not using them
- you are defining your own imported targets  (by using `libgrpc STATIC IMPORTED` and specifying the file path). That way, the metadata of the original grpc targets is not preserved (e.g. the imported targets don't know what their dependencies are, so you need to re-define all of them manually and include them in the helloworld targets)
- I think this should be fixable by getting rid of all the imported targets, and using the `gRPC::xyz`  and `protobuf::xyz` targets instead.